general_configuration/_topics/configuration.md

```diff
@@ -1568,153 +1566,6 @@ To schedule a SmartState Analysis or Compliance Check:
-
-6.  From the **Action** list, select **Database backup**.
-
-7.  In the **Database Backup Settings** area, select a type of server to
-    put the backups. You can either use **Network File System** or
-    **Samba**.
-
-      - If selecting **Samba**, enter the **Depot Name**, **URI**,
-        **User ID**, and a valid **Password**. Then, click **Validate**
-        to check the settings.
-
-      - If you choose **Network File System**, enter the **Depot Name**
-        and **URI**.
-
-8.  In **Run**, set the frequency of the analysis to run. There are
-    further options based on which **Run** option you choose.
-
-      - Click **Once** to have the backup run only one time.
-
-      - Click **Daily** to run the backup on a daily basis. You will be
-        prompted to select the number of days between each backup.
-
-      - Click **Hourly** to run the backup hourly. You will be prompted
-        to select the number of hours between each backup.
-
-9.  Select a **Time Zone**.
-
-    **Note:**
-
-    If you change the **Time Zone**, you will need to reset the stating
-    date and time.
-
-10. Type or select a date to begin the schedule in **Starting Date**.
-
-11. Select a **Starting Time** (UTC) based on a 24 hour clock in the
-    selected time zone.
-
-12. Click **Add**.
-
-###### Modifying a Schedule
-
-To modify a schedule:
-
-1.  Browse to **Settings** > **Application Settings**.
-
-2.  Click on the **Settings** accordion, then click **Schedules**.
-
-3.  Click the schedule that you want to edit.
-
-4.  Click ![1847](../images/1847.png) (**Configuration**), and then click
-    ![1851](../images/1851.png) (**Edit this Schedule**).
-
-5.  Make the required changes.
-
-6.  Click **Save**.
-
-##### Importing and Exporting Schedules
-
-{{ site.data.product.title_short }} provides the ability to import and export
-schedules using the `import_export_schedules.rb` script:
-
-    ./var/www/miq/vmdb/tools/import_export_schedules.rb -h
-
-| Option                              | Description                        |
-| ----------------------------------- | ---------------------------------- |
-| \-u, --user=\<s\>                   | userid (default: admin)            |
-| \-d, --output-dir=\<s\>             | Output directory (default: ./)     |
-| \-s, --schedule=\<s\>               | Schedule name or id                |
-| \-o, --operation=\<s\>              | Export or import (default: export) |
-| \-y, --import-yaml=\<filename/uri\> | Imported yaml                      |
-| \-h, --help                         | Show this message                  |
-
-**Usage Example.**
-
-Import:
-
-    ./var/www/miq/vmdb/tools/import_export_schedules.rb -o import -y ./filename.yaml
-
-Export:
-
-    ./var/www/miq/vmdb/tools/import_export_schedules.rb -s 158
-
-Note that the `import_export_schedules.rb` script works for all types of
-schedules, including:
-
-  - Report
-
-  - Policy
-
-  - Alert
-
-  - SmartState Analysis
-
-  - Database Backup
```

Comment:
I think this is the only line that should be removed in this last section.

Reply:
The test of this from line 1658 to the end is about generically import/export of schedules, of which DB backup is one of them. We removed db backup schedules but the others still exist.